src/main/java/seedu/duke/Duke.java

```diff
@@ -1,39 +1,46 @@
 package seedu.duke;
 
+import seedu.duke.command.Command;
+import seedu.duke.exceptions.unknownCommandException;
 import seedu.duke.parser.Parser;
 import seedu.duke.storage.Storage;
 import seedu.duke.ui.Ui;
 
-import java.util.Scanner;
 
 public class Duke {
+    public static int indexNumber;
+
     private Ui ui;
     private Parser parser;
     private Storage storage;
 
     public Duke() {
+        indexNumber = 0;
         ui = new Ui();
         parser = new Parser();
         storage = new Storage();
     }
 
```

Comment:
I will probably add a startup method here to load the saved files and anything else (eg intro message)